dired-rsync.el

```diff
@@ -145,7 +145,8 @@ ssh/scp tramp connections."
                        (-flatten
                         (list dired-rsync-command
                               dired-rsync-options
-                              src-files dest)))))
+                              (mapcar #'shell-quote-argument src-files)
```

Comment:
Have you actually got a failure case for this? I think the escaping is handled by the shell process and this has regressed my use case.